src/stubs/reactive_stubs_support.h

```diff
@@ -3,7 +3,7 @@
 
 #include "reactive.h"
 
-void reactive_handle_output(io_index output_id, void* data, size_t len);
+void reactive_handle_output(conn_index conn_id, void* data, size_t len);
```

Comment:
Yes, they are both `uint16_t`, so it does not actually change anything, but I wanted to make a distinction between input/output IDs and connection IDs.

In the old implementation those IDs were exactly the same because we had one-to-one relationships between inputs and outputs, but now that we allow many-to-many relationships we need an unique connection ID for each connection.